### DISCUSSION TITLE
Is the 2FA feature ready yet?

### DISCUSSION THREAD

**Discussion by edardev** [supabase/supabase#7919] [Questions] [answered]
Is the 2FA feature ready yet? I am building a system that needs 2FA and was wondering if the GoTrue API has this feature or should I use something else like ORY/Kratos?

**Comment by J0** [comment, marked as answer]
The 2FA feature (via TOTP) is now available for early signup via [this form](https://airtable.com/shrjcc9zi0RduHLIx)
